src/main/java/thePackmaster/powers/instadeathpack/LockPower.java

```diff
@@ -0,0 +1,68 @@
+package thePackmaster.powers.instadeathpack;
+
+import com.evacipated.cardcrawl.mod.stslib.powers.interfaces.NonStackablePower;
+import com.megacrit.cardcrawl.actions.common.GainBlockAction;
+import com.megacrit.cardcrawl.actions.common.RemoveSpecificPowerAction;
+import com.megacrit.cardcrawl.actions.utility.UseCardAction;
+import com.megacrit.cardcrawl.cards.AbstractCard;
+import com.megacrit.cardcrawl.core.AbstractCreature;
+import com.megacrit.cardcrawl.core.CardCrawlGame;
+import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
+import com.megacrit.cardcrawl.helpers.FontHelper;
+import com.megacrit.cardcrawl.powers.AbstractPower;
+import thePackmaster.SpireAnniversary5Mod;
+import thePackmaster.powers.AbstractPackmasterPower;
+
+
+public class LockPower extends AbstractPackmasterPower implements NonStackablePower {
```

Comment:
Would be nice if this could use Nelly's new icon system (like KillQueen) to show the card it's on.

Reply:
👍